pkg/util/redact/redact.go

```diff
@@ -172,3 +173,34 @@ func DeRedact(remove bool, input io.Reader, output io.Writer, sep string) error
 
 	return nil
 }
+
+// InitRedact inits the enableRedactLog
+func InitRedact(redactLog bool) {
+	mode := errors.RedactLogDisable
+	if redactLog {
+		mode = errors.RedactLogEnable
+	}
+	errors.RedactLogEnabled.Store(mode)
+}
+
+// NeedRedact returns whether to redact log
+func NeedRedact() bool {
+	mode := errors.RedactLogEnabled.Load()
+	return mode != errors.RedactLogDisable && mode != ""
+}
+
+// Value receives string argument and return omitted information if redact log enabled
+func Value(arg string) string {
```

Comment:
I prefer this function to be renamed to `String` to indicate its argument is string type, more like zap.String.

Reply:
this package already have a `String` with quite different impl

this method is actually redacting values inside SQL, i.e. convert values into `?`, so name it as `Value`